backend/src/models/userModel.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-use-before-define */
```

Comment:
Remove eslint comments.

Reply:
Fixed, It was accessing User Model before Modes was instantiated, now I am accessing User using this